DFE-Digital/early-careers-framework
# [CPDLP-845] Update ECF statement dates

## Ticket and context

Ticket: https://dfedigital.atlassian.net/browse/CPDLP-845

- Update the dates present on the current ECF statement

## Tech review

### Is there anything that the code reviewer should know?

### Code quality checks

### HTML Checks

### Gotchas

## Product review

### How can someone see it it review app?

- https://ecf-review-pr-1638.london.cloudapps.digital
- Login as a finance user
- View an ECF payment breakdown
- Should see correct dates

## External API changes

- None
